pubsub/gossipsub/gossipsub-v1.3.md

```diff
@@ -0,0 +1,137 @@
+# gossipsub v1.3: Extensions Control Message
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2025-06-23  |
+
+Authors: [@marcopolo]
+
+Interest Group: TODO
```

Comment:
Please comment here if you'd like to be included in this interest group

Reply:
✋🏽

Reply:
✋🏽

Reply:
me too please